Cloud Application Manager/Billing/billing-menu.md

```diff
@@ -34,21 +34,29 @@ All users with Cloud Application Manager organization administrator access.
 * The user should be at the organization level scope to access the Billing option in the left side menu.
 
 ### Billing Dashboard
-When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the latest months will be shown. You can also change the range of months being shown in the bar graph by selecting a different value from the top-right dropdown on the graph.
+When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the previous months will be shown. You can change the range of months being shown in the bars graph by selecting a different value in the date filter and you can also select the usage types you want to display with the content type filter..
 
 ![Billing Dashboard](../../images/cloud-application-manager/billing/billing-dashboard.png)
```

Comment:
Please, update the screenshot once the labels have been updated (CAM-9106)

Reply:
I have made it